inspirehep/modules/records/jsonschemas/records/hep.json

```diff
@@ -369,7 +338,7 @@
                         "type": "string"
                     },
                     "cnum": {
-                        "type": "array"
+                        "type": "string"
```

Comment:
If the same talk is given at several conferences we usually have only one HEP record. In that case we can also have several pub notes. Currently the CNUM is often linked to the pubnote - they are in the same 773 field. Is that relation kept in the new data model?

- Annette


On May 24, 2016, at 2:22 PM, Samuele Kaplun <notifications@github.com<mailto:notifications@github.com>> wrote:


In inspirehep/modules/records/jsonschemas/records/hep.json<https://github.com/inspirehep/inspire-next/pull/1143#discussion_r64380467>:

> @@ -369,7 +338,7 @@
>                          "type": "string"
>                      },
>                      "cnum": {
> -                        "type": "array"
> +                        "type": "string"


@annetteholtkamp<https://github.com/annetteholtkamp> this is weird though. Is it due to a bug rather than a feature? How can a document be referring to more than 1 conference? (e.g. http://inspirehep.net/record/1405352 refers to 2 conferences separated by several years!)

—
You are receiving this because you were mentioned.
Reply to this email directly or view it on GitHub<https://github.com/inspirehep/inspire-next/pull/1143/files/2702183102d64e73b30914f157e506cf76d1d92f#r64380467>